src/pathfinder/yapf/yapf_road.cpp

```diff
@@ -390,8 +390,7 @@ class CYapfFollowRoadT
 			while (pNode->m_parent != nullptr) {
 				steps--;
 				if (pNode->GetIsChoice() && steps < YAPF_ROADVEH_PATH_CACHE_SEGMENTS) {
-					TrackdirByte td;
-					td = pNode->GetTrackdir();
+					Trackdir td = pNode->GetTrackdir();
 					path_cache.td.push_front(td);
```

Comment:
No need even for `Trackdir td` now, `push_front(pNode->GetTrackdir())` will suffice.